openshift-knative-operator/hack/update-manifests.sh

```diff
@@ -37,8 +41,32 @@ function download {
   done
 }
 
+function download_ingress {
```

Comment:
Why is the `net-istio` download inside the `openshift-knatve-operator`, while `kourier` is in the `knative-operator`?

Reply:
Because we didn't yet do the work to migrate the Kourier installation over. The Ingress API of the operator is quite new and us using Kourier predates it's existence. Moving Kourier over is also slightly non-trivial as we install it in `knative-serving-ingress`, while upstream assumes it's installed into `knative-serving`. We have a plan to move that over, but it's not quite necessary for the purposes of this work.

An additional mention: This means that net-kourier will **always** be installed (even if Istio is enabled), and I think that's okay for now, since we're shadow-launching this. We'll just tell people that use Istio that Kourier is still installed but won't work as expected anymore.

Reply:
thanks